gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -19,6 +19,7 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
+    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
```

Comment:
https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

> The target directory is used to house all output of the build.